tests/css_case/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -343,7 +343,7 @@ $KeepScssVar: val;
 
   @{Keep}[@{Keep}][@{Keep}^=@{Keep-1A}]:@{Keep} {
     prop: val;
-  }
+  };
```

Comment:
Because parse as Less `detached ruleset`, but it is invalid syntax for CSS/Less/SCSS, using `@{var}` in Less allowed only in string - i.e. `"@{var}"`